ocs_ci/ocs/resources/mcg.py

```diff
@@ -17,7 +19,11 @@ class MCG(object):
     Wrapper class for the Multi Cloud Gateway's S3 service
     """
 
-    s3_resource, ocp_resource, endpoint, region, access_key_id, access_key, namespace = (None,) * 7
+    (
```

Comment:
Shouldn't we do this initialization inside an __init__() func?

Reply:
Strictly speaking, there's no need at all to declare those valuables in Python - it's mostly for documentation and readability. We can take this off completely - I tend to prefer leaving it out.

Reply:
Apart from readability, I am too not aware of any other reason. I'm ok with both